src/components/Reports/Reports.jsx

```diff
@@ -1,17 +1,223 @@
+    this.setState((state) => {
+      return {
+        checkActive:'true'
+      }
+    });
+  }
+
+  setAll() {
+    this.setState((state) => {
+      return {
+        checkActive:''
+      }
+    });
+  }
+
+  setInActive(){
+
+    this.setState(()=>({
+      checkActive:'false'
+    }))
+  }
+
+  showProjectTable() {
+    this.setState(prevState => ({
+      showProjects: !prevState.showProjects,
+      showPeople: false,
+      showTeams: false,
+      // showTasks: false
+    }))
+  }
+
+  showTeamsTable() {
+    this.setState(prevState => ({
+      showProjects: false,
+      showPeople: false,
+      showTeams: !prevState.showTeams,
+    }))
+  }
+
+  showPeopleTable() {
+    this.setState(prevState => ({
+      showProjects: false,
+      showPeople: !prevState.showPeople,
+      showTeams: false,
+    }))
+      }
+
+
+  showTasksTable(){
+    this.setState(prevState => ({
+      showProjects: false,
+      showPeople: false,
+      showTeams: false,
+    }))
+  }
   render() {
+    let { projects} = this.props.state.allProjects;
+    let { allTeams } = this.props.state.allTeamsData;
+    let { userProfiles } = this.props.state.allUserProfiles;
+
+
+
+    if (this.state.checkActive ==='true'){
+      projects = projects.filter(project => project.isActive ===true);
+      userProfiles =userProfiles.filter(user => user.isActive ===true);
+      allTeams =allTeams.filter(team => team.isActive ===true);
+    }
+    else if (this.state.checkActive ==='false'){
+      projects = projects.filter(project => project.isActive ===false);
+      userProfiles =userProfiles.filter(user => user.isActive ===false);
+      allTeams =allTeams.filter(team => team.isActive ===false);
+    }
+
+
     return (
       <div>
-        <div className="jumbotron">ReportsPage</div>
         <div>
           <nav className="navbar navbar-expand-md navbar-light bg-light mb-3 nav-fill">
             <li className="navbar-brand">Generate Report:</li>
           </nav>
+          <a>Select a Category</a>
+          <button style={{margin:'3px'}} exact className="btn btn-secondary btn-bg mt-3" onClick={this.showProjectTable}>Project</button>
+          <button style={{margin:'3px'}} exact className="btn btn-secondary btn-bg mt-3" onClick={this.showPeopleTable}>Person</button>
+          <button style={{margin:'3px'}} exact className="btn btn-secondary btn-bg mt-3" onClick={this.showTeamsTable}>Team</button>
+          <div>
+          <a>Select a Filter</a>
+            <input name='radio' type="radio" style={{margin:'5px'}} value="active" onChange={this.setActive}  />
+            Active
+            <input name='radio' type="radio" style={{margin:'5px'}} value="inactive" onChange={this.setInActive } />
+            InActive
+            <input name='radio' type="radio" style={{margin:'5px'}} value="all" onChange={this.setAll }defaultChecked />
+            All
+          <button style={{margin:'5px'}} type="submit" className="btn btn-primary btn-bg mt-3">
```

Comment:
You should try to put the submit button on a  new line you can look at reactstrap examples to get that to work or can just make another div.